features/api/cves.feature

```diff
@@ -0,0 +1,318 @@
+Feature: Client behaviour for CVE vulnerabilities API
+
+  @uses.config.contract_token
+  Scenario Outline: CVE vulnerabilities for xenial machine
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+    When I attach `contract_token` with sudo
+    # Check we can download and parse the JSON data
+    And I run `pro api u.pro.security.cves.v1` as non-root
+    And I verify that running `pro api u.pro.security.cves.v1 --data '{"fixable": true, "unfixable": true}'` `as non-root` exits `1`
+    Then API errors field output is:
+      """
+      [
+        {
+          "code": "invalid-option-combination",
+          "meta": {
+            "option1": "unfixable",
+            "option2": "fixable"
+          },
+          "title": "Error: Cannot use unfixable together with fixable."
+        }
+      ]
+      """
```

Comment:
Well I see the point, but would it make sense for a user perspective to just treat it as both and show them all?
I mean, if we are showing all by default and someone asks for fixable and unfixable, that's not conflicting, but rather complementary. I checked the spec, there is no special case defined for this.

Reply:
Good point, I am now supporting both flags together